src/index.js

```diff
@@ -482,34 +481,13 @@ class AvatarEditor extends React.Component {
         y: lastY
       } = this.getCroppingRect()
 
-      lastX *= width
```

Comment:
Just pulled the branch and panning is currently broken due to the removal of 485-6 :)